src/platform/plugins/private/presentation_panel/public/panel_component/use_panel_error_css.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { useEuiTheme } from '@elastic/eui';
```

Comment:
nit: why a new file for this when it could be inlined in `presentation_panel_internal`? We often inline emotion styles.

Reply:
Its used in `presentation_panel_internal` and `presentation_panel`